app/src/main/java/org/jellyfin/mobile/fragment/ConnectFragment.kt

```diff
@@ -149,13 +150,18 @@ class ConnectFragment : Fragment() {
         chooseServerButton.isVisible = false
     }
 
-    private fun showConnectionError(@StringRes errorString: Int = R.string.connection_error_cannot_connect) {
-        connectionErrorText.setText(errorString)
-        connectionErrorText.isVisible = true
+    private fun showConnectionError(error: String? = null) {
+        connectionErrorText.apply {
+            if (error != null) text = error else setText(R.string.connection_error_cannot_connect)
```

Comment:
```suggestion
           text =  if (error != null) error else getString(R.string.connection_error_cannot_connect)
```

Reply:
Makes it easier to read imo

Reply:
Not the same though :wink:
But in our case, probably should be.

Reply:
oh yeah `?:` is a thing